### PR TITLE
Address #131. Clarify payment use case.

### PR DESCRIPTION
**Pull request by zolkis**
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -918,15 +918,21 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the payment process itself, but recording the fact of payment with a web
```
Comment (kenchris) [inline index.html]: recording the fact of payment - is a bit hard to understand

Comment (zolkis) [reply index.html]: suggest

File: index.html [hunk 2]
```diff
@@ -918,15 +918,21 @@
...
         device and the point of sales terminal. With the Web NFC API, if the
         user navigates to a web site before paying, there may be interaction
         with that site regarding the payment, e.g. the user could get points and
         discounts, or get delivered application or service specific data (e.g.
         tickets, keys, etc) to the device.
-        This use case is not supported in this version of the specification.
+      </p>
+      <p class="note">
+        This use case is not supported in this version of the specification (see
+        also <a href="https://github.com/w3c/web-nfc/issues/131">issue 136</a>).
```
Comment (kenchris) [inline index.html]: remove also

File: index.html (hunk 1 above)
Comment (zolkis) [reply index.html]: the example should clarify

File: index.html (hunk 2 above)
Comment (zolkis) [reply index.html]: done

Comment (zolkis) [reply index.html]: although issue 131 is not a full coverage of not supporting payment use cases in the spec

File: index.html (hunk 1 above)
Comment (kenchris) [reply index.html]: I am trying to wrap my head around what you wanted to say :)

"but the fact that a payment on the web may happen by the use of NFC technology, which adds the ability to perform secondary NFC actions at the same time"

Comment (zolkis) [reply index.html]: > "but the fact that a payment on the web may happen by the use of NFC technology, which adds the ability to perform secondary NFC actions at the same time"

now it's me who doesn't understand your sentence :D

@anssiko chime in?

Comment (anssiko) [reply index.html]: How about:

NFC can have multiple records that are read when touched, and Web NFC actions can be taken at the same time with the payment that may be implemented using native NFC technology.

Comment (anssiko) [reply index.html]: @zolkis please update the PR if this works for you, @kenchris approved the proposed text.

Comment (zolkis) [reply index.html]: Amended again, please check if it's better than now. The use case text is written in the context of host card emulation with secure element.

File: index.html [hunk 3]
```diff
@@ -918,15 +918,21 @@
...
         In general, touching the device to the point of sales terminal receiver
         area will result in a transaction between the secure element from the
         device and the point of sales terminal. With the Web NFC API, if the
         user navigates to a web site before paying, there may be interaction
         with that site regarding the payment, e.g. the user could get points and
         discounts, or get delivered application or service specific data (e.g.
         tickets, keys, etc) to the device.
-        This use case is not supported in this version of the specification.
+      </p>
+      <p class="note">
```
Comment (anssiko) [inline index.html]: I suggest you make this note simply:

>This use case is not supported in this version of the specification.

Comment (zolkis) [reply index.html]: ok

File: index.html [hunk 4]
```diff
@@ -918,14 +918,19 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the NFC payment process itself, but associating the payment status with
```
Comment (alexshalamov) [inline index.html]: Maybe rephrase it differently? NFC can be used as a 'companion' technology for payments, e.g., for EMV, thus, NFC is not a payment process.

> Payment scenarios with Web NFC generally do not refer to supporting the NFC payment process itself

Payment scenarios with Web NFC generally do not refer to supporting the payment process itself

File: index.html [hunk 5]
```diff
@@ -918,14 +918,19 @@
...
+        a web page in order to have secondary actions. For instance,
+        the user buys goods in a store, and payments options include NFC.
```
Comment (alexshalamov) [inline index.html]: include contactless payment using NFC technology?

**Review by alexshalamov — APPROVED** [review]
few nits

File: index.html (hunk 5 above)
Comment (zolkis) [reply index.html]: Thanks! Amended for the new changes.

**Review by alexshalamov — APPROVED** [review]

**Comment by anssiko** [comment]
Thanks for the contributions, @zolkis you can merge this now.

We should keep on maintaining the use cases document along the specification. The use cases doc is to be considered a "living document". It is normal that the initial assumptions made in the beginning may change as we collectively gain more insight.
